psi4/src/read_options.cc

```diff
@@ -248,6 +248,40 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     /*- PCM-CCSD algorithm type. -*/
     options.add_str("PCM_CC_TYPE", "PTE", "PTE");
   }
+  
+  /*- PE boolean for polarizable embedding module -*/
+  options.add_bool("PE", false);
+  if (name == "PE" || options.read_globals()) {
+    /*- MODULEDESCRIPTION Performs polarizable embedding model (PE) computations. -*/
+
+    /*- Name of the potential file -*/
+    options.add_str_i("POTFILE", "potfile.pot");
+    /* - Use DIIS acceleration to obtain induced moments -*/
+    options.add_bool("DIIS", true);
+    /* - Threshold for induced moments convergence -*/
+    options.add_int("CONVERGENCE_INDUCED", 8);
+    /* - Maximum number of iterations for induced moments -*/
+    options.add_int("MAXITER", 50);
+    
+    /* - Activate border options for sites in proximity to the QM/MM border -*/
+    options.add_bool("BORDER", false);
+    /* - border type, either remove or redistribute moments/polarizabilities -*/
+    options.add_str_i("BORDER_TYPE", "REMOVE", "REMOVE REDIST");
```

Comment:
Avoid the case-preserving `str_i` except where it's really needed, like filenames.